app/code/Magento/CatalogImportExport/Test/Integration/_files/overrides.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<overrides>
```

Comment:
We didn't make changes in "Magento\CatalogImportExport\Model\ProductTest" in that PR.
Why we need skip that? I think these override can be deleted